src/settings-ui/Microsoft.PowerToys.Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1706,4 +1709,10 @@ From there, simply click on a Markdown file, PDF file or SVG icon in the File Ex
     <value>Do not activate when Game Mode is on</value>
     <comment>"Game mode" is the Windows feature to prevent notification when playing a game.</comment>
   </data>
+  <data name="FancyZones_Radio_Custom_Theme.Content" xml:space="preserve">
+    <value>Custom</value>
```

Comment:
Let's use `custom color` to make the context more clear. it has nothing to do with opacity.

Reply:
But you're updating the Zone appearance.. so Custom makes sense then (this is also what W11 does).

Reply:
Sure. But switching the drop down only show/hides the color properties and has no effect on the opacity setting in this expander.

Reply:
Since now the setting refer to zone appearance I am fine with:
- Custom zone appearance 
- Windows default zone appearance

Reply:
So, what's the issue here?
Does the drop down include the opacity or not?
If not, I assume Custom Colors does make more sense.

Reply:
No it doesn't include.